lib/watir/elements/select.rb

```diff
@@ -198,6 +250,33 @@ def matches_regexp?(how, element, exp)
         raise Error, "unknown how: #{how.inspect}"
       end
     end
+
+    def text_script
+      <<SCRIPT
```

Comment:
I propose to make these atoms or at least put into separate file and read on load. E.g. like https://github.com/p0deje/watir-dom-wait/blob/master/lib/watir/dom/elements/element.rb#L3